lb_content_resolver/unresolved_recording.py

```diff
@@ -42,13 +42,33 @@ def add(self, recording_mbids):
             of times it has been unresolved.
         """
 
-        query = """INSERT INTO unresolved_recording (recording_mbid, last_updated, lookup_count)
-                        VALUES (?, ?, 1)
-         ON CONFLICT DO UPDATE SET lookup_count = EXCLUDED.lookup_count + 1"""
+        recording_mbids = list(set(recording_mbids))
+
+        placeholders = ",".join(("?", ) * len(recording_mbids))
+        existing = {
+                row[0]:True for row in db.execute_sql(
```

Comment:
```suggestion
                row[0] for row in db.execute_sql(
```

Why a dict here? A set would work the same as `existing` is only used below in `if mbid in existing:`

Reply:
A dict is an index, thus faster than sequentially scanning a list.